deploy-agent/tests/unit/deploy/server/test_agent.py

```diff
@@ -545,5 +545,31 @@ def test_send_deploy_status_with_errors(self, mock_create_sc):
         self.assertEqual(agent._curr_report.report.deployStage, DeployStage.PRE_DOWNLOAD)
         self.assertEqual(agent._curr_report.report.status, AgentStatus.UNKNOWN)
 
+    @mock.patch('deployd.agent.create_sc_increment')
+    def test_send_deploy_status_with_telefig_unavailable_errors(self, mock_create_sc):
+        status = DeployStatus()
+        status.report = PingReport(jsonValue=self.deploy_goal1)
+
+        envs = {'abc': status}
+        client = mock.Mock()
+        estatus = mock.Mock()
+        estatus.load_envs = mock.Mock(return_value=envs)
+        ping_response_list = [
+            PingResponse(jsonValue=self.ping_response1),
+            PingResponse(jsonValue=self.ping_noop_response)
+        ]
+        mock_create_sc.return_value = None
+
+        client = mock.Mock()
+        client.send_reports = mock.Mock(side_effect=ping_response_list)
+        agent = DeployAgent(client=client, estatus=estatus, conf=self.config,
+                            executor=self.executor, helper=self.helper)
+        report = DeployReport(status_code=AgentStatus.UNKNOWN, error_code=1, output_msg="/usr/local/bin/configure-serviceset: No such file or directory", retry_times=3)
```

Comment:
if you choose to put TELEFIG_BINARY in config, then you can reference it here, to avoid 2 hardcoded values relying on each other.

Reply:
We can refer the const in common utils here as well but it's a mocking value of an error message. We probably would not want to build a mock message based on the logic of the tested function.

Reply:
yeah, i have seen what you are referring to before, it's called "ugly mirror".  I am just referring to the telefig value, not the whole message.